lighthouse-core/config/config.js

```diff
@@ -340,27 +339,50 @@ class Config {
     return config;
   }
 
- /**
-  * Return IDs of top-level aggregations from a config. Used by tools driving lighthouse-core
-  * @param {{aggregations: !Array<{name: string}>}} Lighthouse config object.
-  * @return {!Array<string>}  Name values of aggregations within
-  */
-  static getAggregations(config) {
-    return config.aggregations.map(agg => ({
-      name: agg.name,
-      id: agg.id
-    }));
+  /**
+   * @param {!Object<{audits: !Array<{id: string}>}>} categories
+   * @param {Array<string>=} categoryIds
+   * @param {Array<string>=} auditIds
+   * @return {!Object<{audits: !Array<{id: string}>}>}
+   */
+  static filterCategoriesAndAudits(categories, categoryIds = [], auditIds = []) {
```

Comment:
can you remove the `auditIds` stuff for now and move it to the PR that uses it/tests it? Also different variable names for parameter `categories` vs new `categories` would be good here too